src/LinuxTracing/PerfEventOpen.h

```diff
@@ -91,22 +91,6 @@ static constexpr uint64_t SAMPLE_REGS_USER_SP_IP_ARGUMENTS =
     (1lu << PERF_REG_X86_DI) | (1lu << PERF_REG_X86_SP) | (1lu << PERF_REG_X86_IP) |
     (1lu << PERF_REG_X86_R8) | (1lu << PERF_REG_X86_R9);
 
-// Max to pass to perf_event_open without getting an error is (1u << 16u) - 8,
-// because the kernel stores this in a short and because of alignment reasons.
-// But the size the kernel actually returns is smaller, because the maximum size
-// of the entire record the kernel is willing to return is (1u << 16u) - 8.
-// If we want the size we pass to coincide with the size we get, we need to pass
-// a lower value. For the current layout of perf_event_stack_sample, the maximum
-// size is 65312, but let's leave some extra room.
```

Comment:
Could we still have this comment (possibly adapted) somewhere? Maybe around `stack_sample_event_open`?

Reply:
Ah, I see you added it in `ClientFlags.cpp`. I would prefer to have it closer to the `perf_event_open` stuff, and then maybe a briefer comment in `ClientFlags.cpp`.